rllib/BUILD

```diff
@@ -1818,14 +1818,21 @@ py_test(
     srcs = ["core/rl_module/tests/test_marl_module.py"]
 )
 
-
+# to be removed
```

Comment:
what is this? if it's a TODO, please add TODO and your handle.

Reply:
yes sir. Fixed.